uit_plus_job/models.py

```diff
@@ -236,6 +236,7 @@ class UitPlusJob(PbsScript, TethysJob):
     _remote_workspace = models.TextField(blank=True)
     _remote_workspace_id = models.CharField(max_length=100)
     qstat = JSONField(null=True)
+    archived = models.BooleanField(default=False)
```

Comment:
This change will require a migration. You may have already generated it. Can you include that migration as part of the PR?